shared/locales/src/nl.json

```diff
@@ -180,9 +180,29 @@
         },
         "Groups": {
             "review": {
-                "title": "Kijk de leeftijdgroepen na",
+                "title": "Kijk de groepen na",
                 "description": "Kijk of de prijzen, de leeftijd en de beschrijving correct zijn ingesteld."
             }
+        },
+        "Functions": {
+            "review": {
+                "title": "Kijk de functies na",
+                "description": "Kijk de toegansrechten en functies na."
+            },
+            "todo": {
+                "title": "Voeg functies toe",
+                "description": "Voeg een hoofdbeheerder toe."
+            }
+        },
+        "Companies": {
+            "review": {
+                "title": "Kijk de facturatiegegevens na",
+                "description": "Kijk na of de facturatiegegevens correct zijn."
+            },
+            "todo": {
+                "title": "Voeg facturatiegegevens toe",
+                "description": "Voeg minstens één juridische entiteit toe."
+            }
```

Comment:
Best in de toekomst enkel keys toevoegen zonder objects, zoals `"Functions.review.title": "waarde"` - dat gaat er in de toekomst uit